tests/src/Traits/OgTrait.php

```diff
@@ -113,4 +114,46 @@ protected function getOgRoles(array $roles, EntityInterface $group) {
     return array_values(OgRole::loadMultiple($ids));
   }
 
+  /**
+   * Checks if the given content belongs to the given parent rdf entity.
+   *
+   * If there are multiple entities or parents with the same title, then
+   * only the first one is checked.
+   *
+   * @param string $parent
+   *   The name of the parent.
+   * @param string $parent_type
+   *   The bundle of the parent.
+   * @param string $content_type
+   *   The bundle of the content.
+   * @param string $content
+   *   The title of the content.
+   *
+   * @throws \Exception
+   *   Thrown when a event with the given title does not exist.
+   */
+  public function assertOgMembership($parent, $parent_type, $content_type, $content) {
```

Comment:
I would uniform the order of the arguments: now it's title, bundle, bundle, title. I suggest bundle, title, bundle, title.